group-coordinator/src/main/resources/common/message/ConsumerGroupMemberMetadataValue.json

```diff
@@ -35,6 +35,13 @@
     { "name": "RebalanceTimeoutMs", "type": "int32", "versions": "0+", "default": -1,
       "about": "The rebalance timeout" },
     { "name": "ServerAssignor", "versions": "0+", "nullableVersions": "0+", "type": "string",
-      "about": "The server assignor to use; or null if not used." }
+      "about": "The server assignor to use; or null if not used." },
+    { "name": "SupportedProtocols", "type": "[]ClassicJoinGroupRequestProtocol", "versions": "0+",
+      "about": "The list of protocols that the member supports if the consumer uses the legacy protocol.", "fields": [
+      { "name": "Name", "type": "string", "versions": "0+", "mapKey": true,
+        "about": "The protocol name." },
+      { "name": "Metadata", "type": "bytes", "versions": "0+",
+        "about": "The protocol metadata." }
+    ]}
```

Comment:
I think that it may be better to introduce a struct field `ClassicMemberMetadata` which then contain the `SupportedProtocols` field. I would also use a nullable tagged field for `ClassicMemberMetadata` in order to make it optional. What do you think?